examples/user-management/refine-user-management/.prettierrc

```diff
@@ -0,0 +1 @@
+{ "tabWidth": 2 }
```

Comment:
Do we need this file? Its kinda strange for an example

Reply:
Removed it.